setup.py

```diff
@@ -126,7 +126,7 @@ def run(self):
 
         "psutil == 2.1.2",
         "netifaces >= 0.8",
-        "ipaddr == 2.1.10",
+        "ipaddr == 2.1.11",
```

Comment:
I think the reason is that `ipaddr` used to link to their external google code download URL...which is frowned upon. 2.1.11 seems to be the first version where they've started properly hosting the package on PyPI and *properly* updating their metadata.

Here's a comment on the subject:

> There are some more notable names in the "unsafe" lists, but a few
spot checks on projects like PyGObject, PyGTK, biopython, dbus-python,
django-piston, ipaddr, matplotlib, and mayavi showed that a number of
them *have* switched to PyPI hosting for recent releases, but have
left older releases as externally hosted. (A few notable names, like
wxPython and Spyder, *did* show up as genuinely externally hosted.
Something that would be nice to be able to do, but isn't really
practical without a server side dependency graph, is to be able to
figure out how many packages have an externally hosted dependency
*somewhere in their dependency chain*, and *how many* other projects
are depending on particular externally hosted projects transitively).

https://mail.python.org/pipermail/distutils-sig/2014-May/024238.html

So I think you're doing the right thing by incrementing the version number here.